setup/sql/dbupdate_custom.php

```diff
@@ -339,3 +339,8 @@
 <?php
 $ilCtrlStructureReader->getStructure();
```

Comment:
leave this out.
Otherwise, it will re-read the structure twice upon fresh installation.

Reply:
It won't. ILIAS writes only a ini value to 1 within this function and at the end of the update it reloads only once.

Reply:
Und wieder etwas gelernt. OK.